draft-ietf-quic-recovery.md

```diff
@@ -346,9 +346,7 @@ On subsequent RTT samples, smoothed_rtt and rttvar evolve as follows:
 
 ~~~
 ack_delay = min(Ack Delay in ACK Frame, max_ack_delay)
-adjusted_rtt = latest_rtt
-if (min_rtt + ack_delay < latest_rtt):
-  adjusted_rtt = latest_rtt - ack_delay
+adjusted_rtt = max(latest_rtt - ack_delay, min_rtt)
```

Comment:
This is the pseudocode that results from https://github.com/quicwg/base-drafts/issues/3350, so this change is not editorial

Reply:
Right, this is a functional change.

Prior to this you get `latest_rtt` or `latest_rtt - ack_delay`.

With this change you get `min_rtt` or `latest_rtt - ack_delay`.

Reply:
Ups, yes, you are right. I think I got that impression from reading "MUST NOT apply the adjustment if the resulting RTT sample is smaller than the min_rtt", however, that's not what it says.

I think changing the if to the following would be slightly more readable for me but doesn't make a big difference:

if (latest_rtt - ack_delay > min_rtt):

Or even

 adjusted_rtt = latest_rtt  - ack_delay
 if (adjusted_rtt > min_rtt):
   adjusted_rtt = latest_rtt

However, we can also just withdraw this PR.